### DISCUSSION TITLE
NVIDIA 的 GTC 會議開場 I am AI 影片

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#11] [General]
2021年Keynote：[GTC 2021 Keynote with NVIDIA CEO Jensen Huang](https://youtu.be/eAn_oiZwUXA)
2021年I am AI影片：https://youtu.be/zNX1knTo2F4
2020年I am AI影片：https://youtu.be/e2_hsjpTi4w
2019年I am AI影片：https://youtu.be/GWL1HNHDSq4
2018年I am AI影片：https://youtu.be/GiZ7kyrwZGQ
2017年I am AI影片：https://youtu.be/SUNPrR4o5ZA

**Comment by briankuo123** [comment]
當初看到今年的I AM AI的時候很震撼。
AI運用的廣泛程度已經超乎我的想像，尤其是用來幫助視覺的部分我是真的沒有想過。
而原本以為只是普通影片背景音樂，在最後卻得知是由AI創作的，真的會有很重的後勁。
我想不用幾年就會像通過圖靈測試那樣無法分辨身邊的事物到底是人為還是AI的作品。
